gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -160,13 +160,16 @@ public abstract class AnvilExtension @Inject constructor(
    * com.squareup.anvil.useKspComponentMergingBackend=true
    * ```
    *
+   * @param project the Gradle project this is operating on. Annoying but necessary to pass this
```

Comment:
😁

You can make `Project` a receiver, so that it's invisible to the DSL.

Reply:
I thought about that, but are you worried about groovy accessors?

Reply:
I am now 😓